### ISSUE TITLE
Fix a confusion with the comment threads

### ISSUE THREAD

**Issue by tio-trom** [friendica/friendica#13444]
Here's something that for me is not confusing, but I see more and more people being confused. This is a post with comments:
![2023-09-15_13-11](https://github.com/friendica/friendica/assets/38043315/d98fe210-e4a1-421f-9a3b-f8b0a7c6c35f)
People seem to be confused when underneath the comments thread there is a box saying "Reply to..." and mentions the name of the post author.
![2023-09-15_13-13](https://github.com/friendica/friendica/assets/38043315/d3fe1e70-01fa-48d4-823d-698c8dc15836)
They use that to reply to the last comment in the thread it seems. I am personally baffled because I think it is intuitive that this is to reply to the post itself. But when the author is also the last one in the thread, then I see why they get confused.

**Solution:**

For one, do not show the thread comment box unless the users click that from the post reply button itself. And when they click to reply to the post itself, show the comment box bellow the post itself directly.
![2023-09-15_13-18](https://github.com/friendica/friendica/assets/38043315/f8c8d463-f85f-4c69-9696-b51460cbc47a)

Second, perhaps change the reply box text for the thread into "Reply to the post made by X" or something like that. To make sure people understand.

Lastly, for long threads it is great to add a little design change, a line for the indent that connects comments and replies similar to this

![2023-09-15_13-20](https://github.com/friendica/friendica/assets/38043315/c7aad63c-3868-4eff-975d-6548debeeeb2)

Here's a long thread:
![b1a903e6-ef00-4fa6-bb3c-b5dad441e3fe](https://github.com/friendica/friendica/assets/38043315/72bcab1f-2095-428a-81b9-f897a774ce88)

**Comment by MrPetovan** [comment]
Good point about the comment box display. For the indentation, by default we condense conversations when it makes sense to reduce the indentation level, but you can disable this behavior by turning off Smart Threading in Settings -> Display -> Content

**Comment by tio-trom** [comment]
I turned that off but am not sure I see a difference. However that idea with the border/lines that connect the comments could be useful if it is not hard to implement. I was looking at the CSS but as far as I can tell the comments are not tagged as "comments" but "media posts" or "posts", like all other posts. So I am not sure it can simply be achieved with CSS.

**Comment by MrPetovan** [comment]
Comments are included in each other in the HTML, so it's definitely possible. Someone tried it and submitted POC screenshots, I can't remember who or when though 😞

**Comment by MrPetovan** [comment]
> I turned that off but am not sure I see a difference.

Smart threading flattens the parts of the conversation where people are replying to each other in sequence. So instead of 

- Author 1
  - Author 2
    - Author 1
      - Author 2

We flatten it to:

- Author 1
- Author 2
- Author 1
- Author 2

It gets a little bit more complicated when there's a third author but it should correctly show when the conversation actually branches.

**Comment by tio-trom** [comment]
Thanks!

That's what I was expecting but with that off I can still see indentation. https://social.trom.tf/display/dbc8dc44-1565-021e-c1ad-a61097014720

Does it take time to "apply"?

**Comment by MrPetovan** [comment]
No, and here are the expectations:
- Smart Threading disabled: Full indentation and all the replies to the same messages are on the same level. Chains of 1-to-1 replies create indentation stairs.
- Smart Threading enabled: Indentation is reduced to only a meaningful amount. Stairs are flattened.

Since I'm anonymous I see the conversation you linked to with Smart Threading on since it's enabled by default.
